server/channels/utils/subpath.go

```diff
@@ -113,15 +113,15 @@ func updateRootFile(oldRootHTML string, rootHTMLPath string, alreadyRewritten bo
 	// be updated (and isn't covered by the cases above).
 	newRootHTML = strings.Replace(newRootHTML, pathToReplace, newPath, -1)
 
+	publicPathInWindowsScriptRegex := regexp.MustCompile(`(?s)<script id="publicPathInWindowScript">(.*?)</script>`)
```

Comment:
I haven't seen that `s` flag before to let the `.` capture newlines. That's really handy, and it feels much nicer than replacing `.` with `[\s\S]` like we've had to do in the web app